ci/env/cleanup_test_state.py

```diff
@@ -0,0 +1,30 @@
+import sys
+
+
+def clear_wandb_project():
+    import wandb
+    from ray.ml.examples.upload_to_wandb import wandb_project
```

Comment:
I'd like to avoid passing any args into the example script, it should be as simple as possible for users to grasp and follow, and looking through argparse does not really contribute to that. The example as is is 3 lines of glue code - can't get much simpler than that.

Reply:
Got it, yep making the example really simple sounds good.

But I'm wondering if there is a way to avoid reading from ray.ml.examples inside a CI script? 

Another option is to keep the example as is, but have a test that calls into `train_model`, and this test can have the project name be configured by env var which is set in the buildkite steps.

Reply:
I'll think about this and consult with @maxpumperla how we could best enable this, especially when moving this to a notebook in docs later on (as the import will not work).
Another option would be to just hard-code it and refer to the example in a comment.